py3status/py3.py

```diff
@@ -482,6 +482,7 @@ def log(self, message, level=LOG_INFO):
                 message = "\n" + message
         except:  # noqa e722
             pass
+        message = message.decode('utf-8')
```

Comment:
I think it should do. Feel free to refactor something else in another PR and close this one if not deemed suitable in some aspects :+1: